engine/mission.py

```diff
@@ -57,6 +58,8 @@ def __init__(self, robot, base_station, init_control_mode, grid=Grid(42.444250,
         self.grid = grid
         self.control_mode = ControlMode(init_control_mode)
         self.all_waypoints = self.grid.get_waypoints(self.control_mode)
+        self.active_waypoints = self.grid.get_active_nodes_list()
```

Comment:
Add documentation for active_waypoints and inactive_waypoints.

Reply:
@Shreya1233